maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapper.java

```diff
@@ -173,7 +173,7 @@ private ApiIncomeEvidence mapToExtraApiIncomeEvidence(ExtraEvidenceDTO evidenceI
         return new ApiIncomeEvidence()
                 .withId(NumberUtils.toInteger(evidenceItem.getId()))
                 .withDateReceived(DateUtil.toLocalDateTime(evidenceItem.getDateReceived()))
-                .withApiEvidenceType(new ApiEvidenceType(evidenceType.getName(), evidenceType.getDescription()))
+                .withIncomeEvidence(evidenceItem.getEvidenceTypeDTO().getEvidence())
```

Comment:
Can you help me understand why we have switched from using the `ApiEvidenceType` to the `IncomeEvidence` type? I thought the `Api` prefixed types were the newer ones that we were moving towards?

Reply:
When we call update financial from ORC to MAAT api, which is return string of incomeEvidence ( FinancialAssessmentDTO.FinAssIncomeEvidenceDTO.incomeEvidence) and it never return type of ApiEvidenceType